Tests/LibAudio/TestWav.cpp

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) 2024, Lee Hanken
```

Comment:
Please include your email address here too.

```suggestion
 * Copyright (c) 2024, Lee Hanken <email@whatever.com>
```

GitHub generates an email for you if you want to keep your real one private.